StarVMC/StarGeometry/CreateGeometry.cxx

```diff
@@ -0,0 +1,75 @@
+#include <TGeoManager.h>
+
+#include "StBFChain/StBFChain.h"
+#include "StarVMC/StarAgmlLib/AgModule.h"
+#include "StarVMC/StarAgmlLib/StarTGeoStacker.h"
+#include "StarVMC/StarGeometry/StarGeo.h"
+
+extern StBFChain* chain;
```

Comment:
```output
dlopen error: /star-sw/.sl79_gcc485/LIB/libStarGeometry.so: undefined symbol: chain
```
Looks like we've never landed the 205f44ac5beac1d92a81f61f6fb8934e4d720f25.

Reply:
Ah, yes! I think that is the reason why half of the tests crashed

Reply:
Now ROOT5 seems to not handle the `extern *chain` in `bfc.C`.